prototype2/adc_readout/test/TestUDPServer.cpp

```diff
@@ -12,13 +12,24 @@
 
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wreorder"
-TestUDPServer::TestUDPServer(std::uint16_t SrcPort, std::uint16_t DstPort, int PacketSize) : Service(), Socket(Service, asio::ip::udp::endpoint(asio::ip::udp::v4(), SourcePort)), Resolver(Service), PacketTimer(Service), SourcePort(SrcPort), DestinationPort(DstPort) {
```

Comment:
I don't care what LLVM says. Wrapping this up in 1 or 2 lines just makes it unreadable...

Reply:
First of all: your code editor sucks. Modern editors have something called "soft line wrap". With that said, I am guessing that code has never been formated by clang-format so doing it like you did is an improvement.